src/Persistence/Initialization/Updates/FixWeaponRisePercentagePlugInSeason6.cs

```diff
@@ -0,0 +1,380 @@
+            [17] = 76,
+            [18] = 92,
+            [19] = 110,
+            [20] = 106,
+            [34] = 130,
+            [36] = 146,
+        };
+        foreach (var stick in sticks)
+        {
+            stick.PossibleItemOptions.Clear();
+            stick.PossibleItemOptions.Add(gameConfiguration.ItemOptions.First(io => io.Name == ExcellentOptions.WizardryAttackOptionsName));
+            stick.PossibleItemOptions.Add(gameConfiguration.ItemOptions.First(io => io.Name == HarmonyOptions.WizardryAttackOptionsName));
+            stick.PossibleItemOptions.Add(gameConfiguration.ItemOptions.First(io => io.PossibleOptions.Any(o => o.OptionType == ItemOptionTypes.Luck)));
+            stick.PossibleItemOptions.Add(gameConfiguration.ItemOptions.First(io => io.PossibleOptions.Any(o => o.OptionType == ItemOptionTypes.Option && o.PowerUpDefinition?.TargetAttribute == Stats.MaximumWizBaseDmg)));
+            if (sticksMagicPower.ContainsKey(stick.Number))
+            {
+                var powerUpDefinition = context.CreateNew<ItemBasePowerUpDefinition>();
+                powerUpDefinition.TargetAttribute = Stats.StaffRise.GetPersistent(gameConfiguration);
+                powerUpDefinition.BaseValue = sticksMagicPower[stick.Number] / 2.0f;
+                powerUpDefinition.AggregateType = AggregateType.AddRaw;
+                powerUpDefinition.BonusPerLevelTable = sticksMagicPower[stick.Number] % 2 == 0 ? staffEvenTable : staffOddTable;
+                stick.BasePowerUpAttributes.Add(powerUpDefinition);
+            }
+        }
+
+        // -> fix Eternal Wing Stick
+        if (weaponsG5.FirstOrDefault(e => e.Number == 20) is { } eternalStick)
+        {
+            eternalStick.DropLevel = 147;
+            var basePowerUps = eternalStick.BasePowerUpAttributes;
+            if (basePowerUps.FirstOrDefault(pu => pu.TargetAttribute == Stats.MinimumPhysBaseDmgByWeapon) is { } minPhysDmgAttr)
+            {
+                minPhysDmgAttr.BaseValue = 66;
+            }
+
+            if (basePowerUps.FirstOrDefault(pu => pu.TargetAttribute == Stats.MaximumPhysBaseDmgByWeapon) is { } maxPhysDmgAttr)
+            {
+                maxPhysDmgAttr.BaseValue = 74;
+            }
+
+            if (basePowerUps.FirstOrDefault(pu => pu.TargetAttribute == Stats.AttackSpeedByWeapon) is { } attackSpeedAttr)
+            {
+                attackSpeedAttr.BaseValue = 30;
+            }
+        }
+
+        // -> fix books
+        if (gameConfiguration.CharacterClasses.FirstOrDefault(cc => cc.Number == (int)CharacterClassNumber.Summoner) is { } summoner)
+        {
+            var bookRiseAttr = context.CreateNew<AttributeDefinition>(Stats.BookRise.Id, Stats.BookRise.Designation, Stats.BookRise.Description);
+            gameConfiguration.Attributes.Add(bookRiseAttr);
+            summoner.AttributeCombinations.Add(CreateAttributeRelationship(context, gameConfiguration, Stats.CurseAttackDamageIncrease, 1.0f / 100, Stats.BookRise));
```

Comment:
I think we will also need to implement `IAssignable` to `AttributeRelationship` like you did [here](https://github.com/MUnique/OpenMU/commit/cc4974fc7e3a56f37b9fb1ee7ce8299167755167) to `AttributeDefinition`, otherwise this won't work.